src/Gui/BitmapFactory.h

```diff
@@ -149,6 +149,8 @@ class GuiExport BitmapFactoryInst : public Base::Factory
     /// Helper method to merge a pixmap into one corner of a QIcon
     static QIcon mergePixmap (const QIcon &base, const QPixmap &px, Gui::BitmapFactoryInst::Position position);
 
+    static qreal getMaximumDPR();

```

Comment:
```suggestion
    static qreal getMaximumDPR() const;
```

Reply:
static member functions can't be marked const.